src/iexamine.cpp

```diff
@@ -101,7 +101,9 @@ void iexamine::cvdmachine( player &p, const tripoint & ) {
 
     // Require materials proportional to selected item volume
     auto qty = loc->volume() / units::legacy_volume_factor;
+    qty = (qty ? qty : 1);
```

Comment:
`qty = std::max( 1, qty )` would be much cleaner.

Reply:
Right, done.